ocs_ci/utility/storage_cluster_setup.py

```diff
@@ -0,0 +1,686 @@
+            if (
+                self.ocs_version >= version.VERSION_4_5
+                and config.ENV_DATA.get("worker_instance_type")
+                == constants.AWS_LSO_WORKER_INSTANCE
+            ):
+                deviceset_data["resources"] = {
+                    "limits": {"cpu": 2, "memory": "5Gi"},
+                    "requests": {"cpu": 1, "memory": "5Gi"},
+                }
+            if (self.ocp_version >= version.VERSION_4_6) and (
+                self.ocs_version >= version.VERSION_4_6
+            ):
+                cluster_data["metadata"]["annotations"] = {
+                    "cluster.ocs.openshift.io/local-devices": "true"
+                }
+            count = config.DEPLOYMENT.get("local_storage_storagedeviceset_count")
+            if count is not None:
+                deviceset_data["count"] = count
+
+        # Allow lower instance requests and limits for OCS deployment
+        # The resources we need to change can be found here:
+        # https://github.com/openshift/ocs-operator/blob/release-4.5/pkg/deploy-manager/storagecluster.go#L88-L116
+        if config.DEPLOYMENT.get("allow_lower_instance_requirements"):
+            none_resources = {"Requests": None, "Limits": None}
+            deviceset_data["resources"] = deepcopy(none_resources)
+            resources = [
+                "mon",
+                "mds",
+                "rgw",
+                "mgr",
+                "noobaa-core",
+                "noobaa-db",
+            ]
+            if self.ocs_version >= version.VERSION_4_5:
+                resources.append("noobaa-endpoint")
+            cluster_data["spec"]["resources"] = {
+                resource: deepcopy(none_resources) for resource in resources
+            }
+            if self.ocs_version >= version.VERSION_4_5:
+                cluster_data["spec"]["resources"]["noobaa-endpoint"] = {
+                    "limits": {"cpu": 1, "memory": "500Mi"},
+                    "requests": {"cpu": 1, "memory": "500Mi"},
+                }
+        else:
+            platform = config.ENV_DATA.get("platform", "").lower()
+            if self.local_storage and platform == "aws":
+                resources = {
+                    "mds": {
+                        "limits": {"cpu": 3, "memory": "8Gi"},
+                        "requests": {"cpu": 1, "memory": "8Gi"},
+                    }
+                }
+                if self.ocs_version < version.VERSION_4_5:
+                    resources["noobaa-core"] = {
+                        "limits": {"cpu": 2, "memory": "8Gi"},
+                        "requests": {"cpu": 1, "memory": "8Gi"},
+                    }
+                    resources["noobaa-db"] = {
+                        "limits": {"cpu": 2, "memory": "8Gi"},
+                        "requests": {"cpu": 1, "memory": "8Gi"},
+                    }
+                cluster_data["spec"]["resources"] = resources
+
```

Comment:
```
        # Disable NFS when cluster resources are insufficient
        from ocs_ci.utility.nfs_utils import check_cluster_resources_for_nfs

        if not check_cluster_resources_for_nfs():
            logger.info(
                "Disabling NFS because cluster resources are insufficient "
                "(based on actual node CPU/memory measurement)"
            )
            cluster_data["spec"]["nfs"] = {"enable": False}

```
We are missing this recent change:
https://github.com/red-hat-storage/ocs-ci/commit/fa047c4430f600ad5f09d43e74fe52eac794d762
Check all changes from PR as there are more lines than above